js/app.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+function getVader() {
+  document.getElementById('person4Name').innerHTML = JSON.parse(this.response).name;
+}
+
+function getTatooine() {
+  document.getElementById('person4HomeWorld').innerHTML = JSON.parse(this.response).name
+}
+let vader = new XMLHttpRequest();
+vader.addEventListener("load", getVader);
+vader.open("GET", "https://swapi.co/api/people/4/");
+vader.send();
+
+let tatooine = new XMLHttpRequest();
```

Comment:
You shouldn't be calling out the planet with a hardcoded url. You should get the url from the person4 response object first and use that to update the planet.